lib/dependenciesInstall.js

```diff
@@ -37,34 +37,23 @@ module.exports = function(context) {
         return Q();
     }
 
-    var dependencies = packageJSON.dependencies;
-
-    var depNames = [];
-    for (var dep in dependencies) {
-        if (dependencies.hasOwnProperty(dep)) {
-            depNames.push(dep);
-        }
-    }
-
-    var packagesToInstall = depNames.map(function(dep) {
-        return dep + '@' + dependencies[dep];
-    });
-
     var npm = context.requireCordovaModule('npm');
     return Q.promise(function(resolve, reject, notify) {
         npm.load({
             loaded: false
         }, function(err) {
             if (err) {
-                console.log('cordova-plugin-livereload: An error occurred while loading npm');
-                reject('cordova-plugin-livereload: npm load failed.');
+                console.log(pluginName + ': An error occurred while loading npm');
+                reject(pluginName + ': npm load failed.');
                 return;
             }
-            npm.commands.install(packageJSONDir, packagesToInstall, function(er, data) {
+	    
+	    // Install the dependencies from package.json
+            npm.commands.install(packageJSONDir, [], function(er, data) { 
                 // log the error or data
                 if (err) {
-                    console.log('cordova-plugin-livereload: dependencies install failed.');
-                    reject('cordova-plugin-livereload: dependencies install failed.');
+                    console.log(pluginName + ': dependencies install failed.');
```

Comment:
`err` might have useful diagnostic information and help us understand what scenarios this might fail in. Please ensure we don't lose it.